lldb/source/API/SBProgress.cpp

```diff
@@ -40,7 +40,10 @@ SBProgress::~SBProgress() = default;
 void SBProgress::Increment(uint64_t amount, const char *description) {
   LLDB_INSTRUMENT_VA(amount, description);
 
-  m_opaque_up->Increment(amount, description);
+  std::optional<std::string> description_opt;
+  if (description && description[0])
```

Comment:
So this only calls increment if there is a non-null and non-empty description. Is that the correct behavior? Or should we still be calling increment with a `nullopt` in that case?

Reply:
No we only set the value of the description optional if non-null, non-empty description. So `Increment(1, NULL/None in Python)` will bump the count by one but send no message.